### ISSUE TITLE
Battery draining

### ISSUE THREAD

**Issue by overkillzworld** [emsec/ChameleonMini#301]
What is the best way to find out why my ChameleonMini always drains the battery even though it is turned off?
Already flashed latest firmware.
Tested different new LIR2032.
Fully charged via USB. 
Power switch is turned off.
After one day it is empty.
Some advice on how to proceed would help me.

**Comment by david-oswald** [comment]
Hi, 

As you can see in the schematics, if the power switch is off the battery is only connected to the charger IC:

![image](https://user-images.githubusercontent.com/5631273/123298473-725fb500-d510-11eb-8259-a0cb7662209d.png)

So I would check how that IC looks / if it gets hot or similar. Another possible issue might be that C4 (parallel to the battery) has failed and conducts (a bit) to ground, do you have a multimeter to check that?

Overall, whether you can repair this yourself would of course depend on your electronics experience.

**Comment by overkillzworld** [comment]
Yes I have a multimeter here. However, I am a developer and not an electronics expert :) What is the best way to measure such a small MLCC? Sorry for the stupid question, but only so I can learn something about it.

**Comment by david-oswald** [comment]
You can take out the battery and then measure the resistance between battery + and - 

This should be high impedance / constantly increase, otherwise there might be an issue with the charging chip or the cap.

**Comment by overkillzworld** [comment]
@david-oswald Thanks for the fast reply.
So it is correct:
Position:
![pos](https://user-images.githubusercontent.com/77024350/123430651-bb734000-d5c8-11eb-9838-02eee95cba38.jpg)
Measure:
![val1](https://user-images.githubusercontent.com/77024350/123430720-cb8b1f80-d5c8-11eb-91ef-99bf9e798f20.jpg)
![val2](https://user-images.githubusercontent.com/77024350/123430760-d80f7800-d5c8-11eb-8e87-2c96fb700498.jpg)
![val3](https://user-images.githubusercontent.com/77024350/123430775-dba2ff00-d5c8-11eb-8304-c60bda28ec79.jpg)

**Comment by david-oswald** [comment]
Yes, so this looks like there's no low impedance in the cap or charging IC - just to make sure, does it go up further than 3.8 M? (this will depend on the multimeter though)

Also, can you check/read the value of R4 on your device?

Finally, have you measured the battery voltage when "fully charged"?

**Comment by timokasper** [comment]
The LIR2032 has 40mAh with 3.6-4.2 Volts. Current=Voltage/Resistance, thus there is a current of about 1 micro Ampere. Thus it takes 40.000 hours / 5 years to drain a full battery ...

so your problem must be elsewhere. 

Are you sure you are using an LIR2032 not a CR2032...     You can measure the voltage at the battery. 4.2 is full, 3.6 is empty, <=3.3 is either a wrong battery (CR2032) or a dead LIR cell

**Comment by overkillzworld** [comment]
> Yes, so this looks like there's no low impedance in the cap or charging IC - just to make sure, does it go up further than 3.8 M? (this will depend on the multimeter though)
> 
> Also, can you check/read the value of R4 on your device?
> 
> Finally, have you measured the battery voltage when "fully charged"?

It stops at 3.85 M Or like seen in the picture 3.89 M.

**Comment by overkillzworld** [comment]
> The LIR2032 has 40mAh with 3.6-4.2 Volts. Current=Voltage/Resistance, thus there is a current of about 1 micro Ampere. Thus it takes 40.000 hours / 5 years to drain a full battery ...
> 
> so your problem must be elsewhere.
> 
> Are you sure you are using an LIR2032 not a CR2032... You can measure the voltage at the battery. 4.2 is full, 3.6 is empty, <=3.3 is either a wrong battery (CR2032) or a dead LIR cell

I am sure it is a LIR2032. Ordered a pack to test if the old LIR was maybe dead. Measured the LIR2032 full charged it has 4.19 V

**Comment by overkillzworld** [comment]
Now that the battery is fully charged, I'll see how long it takes to completely discharge it. Last time it was 1-2 days.

**Comment by overkillzworld** [comment]
I have an Idea what could cause the battery problem, but I wanted to look at the schematic first. Short question offtopic: Which program is best to view the Schematic [.sch] files?  If I try it in KiCad it looks not correct
![sch](https://user-images.githubusercontent.com/77024350/124232640-37621080-db12-11eb-8e49-5eabd8ccd18e.png)

**Comment by david-oswald** [comment]
It's in Eagle, but there is also a PDF in the folder so that you don't have to install that just to check the schematic.

**Comment by overkillzworld** [comment]
Ok. Thanks. I'll take a closer look :)

**Comment by overkillzworld** [comment]
Could it have been a short on the board between C15 and IC2 (or C4 and IC2)?  There the solder joints were a bit close together or slightly connected. I have corrected this and now it looks better since a few days. I'll keep watching it.
